modules/commandModule/commandWorker_taxi_first.py

```diff
@@ -1,25 +1,27 @@
 from modules.commandModule.commandModule import CommandModule
 import logging
+from directories import PIGO_DIR, POGI_DIR
+
 
 def taxi_command_worker_first(pipelineIn, pipelineOut):
-	"""
-	Worker process that gets a turn command [dict] from the search worker and updates the pigo by calling set_ground_command()
-	
-	Parameters
-	----------
-	pipelineIn : multiprocessing.Queue
-	input pipeline that has a turn command dictionary with format {"heading": bearing, "latestDistance": 0}
-	pipelineOut : multiprocessing.Queue
-	output pipeline
+    """
+    Worker process that gets a turn command [dict] from the search worker and updates the pigo by calling set_ground_command()
+
+    Parameters
+    ----------
+    pipelineIn : multiprocessing.Queue
+    input pipeline that has a turn command dictionary with format {"heading": bearing, "latestDistance": 0}
+    pipelineOut : multiprocessing.Queue
+    output pipeline
     
-	Returns
-	-------
-	None
-	"""
-	
-	if pipelineIn.empty():
-		print("No data in taxi_command_worker_first pipelineIn")
-		return
-	
-	turn_command_data = pipelineIn.get()
-	set_ground_commands(turn_command_data)
+    Returns
+    -------
+    None
+    """
+
+    if pipelineIn.empty():
```

Comment:
Should probably just delete this if statement, doing `pipelineIn.get()` on line 26 already handles the empty case for us. I can do this deletion in my own PR and I'll just merge this in for now.